snippets/base/admin/fields.py

```diff
@@ -118,7 +116,9 @@ class JEXLFirefoxRangeField(JEXLRangeField):
     def __init__(self, **kwargs):
         # Include only versions greater than 63, where ASRSnippets exist.
         min_version = 64
-        max_version = int(product_details.firefox_versions['FIREFOX_NIGHTLY'].split('.', 1)[0])
```

Comment:
What makes this method not sufficient?

Reply:
Well, I have some trouble getting this to work nicely on production and/or tests. It should work in theory but in practice it requires that we spend time that we don't really have now, for a little gain. I have #855 to track fixing this in the future.

Reply:
Cool. In that new PR don't forget about the `Version` class in the `product_details` lib.

```python
from product_details.version_compare import Version
from product_details import product_details as pd

max_version = Version(pd.firefox_versions['FIREFOX_NIGHTLY']).major
```

That's the equivalent of what you had but should be more robust. Something to try for later. I agree that doing this simple thing for now that gets this done is good 👍